versioned_sidebars/version-3.9-sidebars.json

```
@@ -197,51 +437,682 @@
+        },
+        {
+          "type": "doc",
+          "id": "scalardl-benchmarks/README",
+          "label": "Run Benchmarks"
+        }
       ]
     },
     {
       "type": "category",
       "label": "Releases",
       "collapsible": true,
       "items": [
-        "releases/release-notes",
-        "releases/release-support-policy"
+        {
+          "type": "doc",
+          "id": "releases/release-notes",
+          "label": "Release Notes"
+        },
+        {
+          "type": "doc",
+          "id": "releases/release-support-policy",
+          "label": "Release Support Policy"
+        }
+      ]
+    }
+  ],
+
+  "docsJapanese": [
+    {
+      "type": "doc",
+      "label": "ScalarDL ドキュメンテーションホーム",
+      "id": "index"
+    },
+    {
+      "type": "category",
+      "label": "ScalarDL について",
+      "collapsible": true,
+      "items": [
+        {
+          "type": "doc",
+          "id": "overview",
+          "label": "概要"
+        },
+        {
+          "type": "doc",
+          "id": "design",
+          "label": "デザイン"
+        },
+        {
+          "type": "doc",
+          "id": "implementation",
+          "label": "実装"
+        },
+        {
+          "type": "doc",
+          "id": "requirements",
+          "label": "要件"
+        },
+        {
+          "type": "doc",
+          "id": "roadmap",
+          "label": "ロードマップ"
+        }
+      ]
+    },
+    {
+      "type": "category",
+      "label": "はじめよう",
+      "collapsible": true,
+      "items": [
+        {
+          "type": "doc",
+          "id": "getting-started",
+          "label": "ScalarDL Ledger を通じてコントラクトを実行"
+        },
+        {
+          "type": "doc",
+          "id": "getting-started-auditor",
+          "label": "ScalarDL Ledger と ScalarDL Auditor を通じてコントラクトを実行"
+        }
+      ]
+    },
+    {
+      "type": "category",
+      "label": "サンプル",
+      "collapsible": true,
+      "items": [
+        {
+          "type": "doc",
+          "id": "applications/simple-bank-account/README",
+          "label": "銀行口座アプリケーション"
+        }
+      ]
+    },
+    {
+      "type": "category",
+      "label": "開発",
+      "collapsible": true,
+      "items": [
+        {
+          "type": "category",
+          "label": "SDKs",
+          "collapsible": true,
+          "items": [
+            {
+              "type": "doc",
+              "id": "scalardl-java-client-sdk/README",
+              "label": "ScalarDL Java クライアント SDK"
+            }
+          ]
+        },
+        {
+          "type": "doc",
+          "id": "configurations",
+          "label": "ScalarDL の設定"
+        },
+        {
+          "type": "doc",
+          "id": "schema-loader",
+          "label": "ScalarDL Schema Loader"
+        },
+        {
+          "type": "doc",
+          "id": "ca/caclient-getting-started",
+          "label": "証明書を取得"
+        },
+        {
+          "type": "doc",
+          "id": "authentication",
+          "label": "認証を使用"
```

```suggestion
          "label": "認証"
```
Same as `sidebars.js`.